src/Fixer/LanguageConstruct/SingleSpaceAroundConstructFixer.php

```diff
@@ -0,0 +1,462 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\LanguageConstruct;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\FixerConfiguration\AllowedValueSubset;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolverInterface;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Preg;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+final class SingleSpaceAroundConstructFixer extends AbstractFixer implements ConfigurableFixerInterface
+{
+    /**
+     * @var array<string, null|int>
+     */
+    private static array $tokenMapContainASingleSpace = [
+        // for now, only one case - but we are ready to extend it, when we learn about new cases to cover
+        'yield_from' => T_YIELD_FROM,
```

Comment:
Shouldn't `const_import` and `function_import` be here?

Reply:
no. they do not contain whitespace.

Reply:
They do? Is it for `use function`, `use const`? 🤔

Reply:
`yield from` is **ONE** token, will be fixed by `contain` option

`use function` are **THREE** tokens, will be fixed by `preceded` / `followed`


ref https://3v4l.org/mW39A